tests/baselines/reference/trailingCommasInFunctionParametersAndArguments.errors.txt

```diff
@@ -0,0 +1,35 @@
+tests/cases/conformance/es7/trailingCommasInFunctionParametersAndArguments.ts(20,11): error TS1138: Parameter declaration expected.
+
+
+==== tests/cases/conformance/es7/trailingCommasInFunctionParametersAndArguments.ts (1 errors) ====

```

Comment:
that's weird, I wasn't expecting to see a .errors.txt and a .types. Usually they are mutually exclusive. Did the .types file manage to skip the broken getter/setter pair?